packages/app/src/app/index.html

```diff
@@ -4,6 +4,8 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- fix cross-origin error, by allowing to run JavaScript from eval or Function constructor source -->
+  <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-eval' 'unsafe-inline' https://*.googleapis.com">
```

Comment:
@lbogdan 
@CompuIves 
Are there any other domain, which are need to load scripts?
Or should be any domain allowed? In that case just `*` would be best.

I have omitted to write a [`default-src` CSP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src), which [should be always provided](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP#Writing_a_policy)...